<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="4">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>List of students</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2" colspan="2">
     <widget class="QPushButton" name="clear">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QPushButton" name="findBest">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Find 3 Best</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="keyLabel">
      <property name="text">
       <string>Student:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Victories:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="3">
     <widget class="QLineEdit" name="victoriesInput"/>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QPushButton" name="remove">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4" rowspan="5">
     <widget class="QTableWidget" name="tableWidget">
      <column>
       <property name="text">
        <string>Student</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Victories</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QLineEdit" name="removeKey"/>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="QLineEdit" name="keyInput"/>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Remove student</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QPushButton" name="addEntry">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Add Entry</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3" colspan="2">
     <widget class="QLabel" name="bestLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
